### DISCUSSION TITLE
File not found error on creating new components

### DISCUSSION THREAD

**Discussion by umresh** [WeblateOrg/weblate#6747] [Q&A] [answered]
Hi,
   I'm new to weblate and while trying to add a component i'm getting this error
<img width="562" alt="Screenshot 2021" src="https://user-images.githubusercontent.com/6183803/140013632-fca060ba-64b0-4b82-bf94-c83bc5253ab3.png">

https://github.com/umresh/Weblate_POC  - Sample Repo 
Assets/Localization/LocalizeData.csv         -  As translation base

Need help how do I solve this? What am I doing wrong?

Thanks.

**Comment by nijel** [comment, marked as answer]
The file name is different in the repo - https://github.com/umresh/Weblate_POC/blob/main/Assets/Localization/Localise%20English%20Data.csv

**Comment by umresh** [comment]
Apologies added wrong screenshot
<img width="620" alt="Screenshot 2021-11" src="https://user-images.githubusercontent.com/6183803/140028958-caed0789-1b0d-46fd-9111-d68043fc957e.png">

Tried multiple times but still getting same error.

**Reply by nijel** [reply]
Are you using the same repo and branch?

**Reply by umresh** [reply]
Repo : https://github.com/umresh/Weblate_POC
branch : develop
File path : Assets/Localization/LocalizeData.csv
Localiza english data.csv is in other branch

<img width="735" alt="Screenshot 2021-11-03" src="https://user-images.githubusercontent.com/6183803/140033709-18416911-d4ce-4811-a348-24af36b3c971.png">

**Reply by nijel** [reply]
...but the file is not present in the develop branch.

**Reply by umresh** [reply]
Oh sorry my bad. Thanks for Quick response.
